Dockerfile

```diff
@@ -16,18 +16,20 @@ RUN mkdir /app
 RUN pip3 install --upgrade pip
 RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
 RUN /root/.local/bin/poetry config virtualenvs.create false
-COPY pyproject.toml /app
-COPY poetry.lock /app
+COPY pyproject.toml poetry.lock /app/
 WORKDIR /app
-RUN /root/.local/bin/poetry install --no-root --extras "all"
+RUN  /root/.local/bin/poetry install --no-root --extras "all" \
+ &&  apt-get purge -y --auto-remove gcc 
 
 COPY src/ /app/src
 
 RUN pip3 install --use-feature=in-tree-build .
-# Install tabulate version that fixes column width wrapping. Cannot be added to pypi as a git dependency, so adding it here
```

Comment:
I'm not totally sure, but I think we need to keep installing tabulate here. We can't upload the `robusta-cli` package to pypi if it has a git dependency in pyproject.toml